packages/logger/test/unit/node.node.test.ts

```diff
@@ -1,27 +1,35 @@
-import {expect} from "chai";
+import {describe, it, expect, vi, afterEach, Mock} from "vitest";
 import {LogLevel} from "@lodestar/utils";
-import {stubLoggerForProcessStd} from "@lodestar/test-utils/mocha";
 import {TimestampFormatCode, logFormats} from "../../src/index.js";
 import {getNodeLogger} from "../../src/node.js";
 import {formatsTestCases} from "../fixtures/loggerFormats.js";
 
+// Node.js maps `process.stdout` to `console._stdout`.
+// spy does not work on `process.stdout` directly.
+// eslint-disable-next-line @typescript-eslint/naming-convention
+type TestConsole = typeof console & {_stdout: {write: Mock}};
+
 describe("node logger", () => {
+  afterEach(() => {
+    vi.resetAllMocks();
+  });
+
   describe("format and options", () => {
     for (const testCase of formatsTestCases) {
       const {id, opts, message, context, error, output} = typeof testCase === "function" ? testCase() : testCase;
       for (const format of logFormats) {
         it(`${id} ${format} output`, async () => {
-          const logger = stubLoggerForProcessStd(
-            getNodeLogger({
-              level: LogLevel.info,
-              format,
-              module: opts?.module,
-              timestampFormat: {format: TimestampFormatCode.Hidden},
-            })
-          );
+          vi.spyOn((console as TestConsole)._stdout, "write");
+
+          const logger = getNodeLogger({
+            level: LogLevel.info,
+            format,
+            module: opts?.module,
+            timestampFormat: {format: TimestampFormatCode.Hidden},
+          });
           logger.warn(message, context, error);
-          logger.restoreStubs();
-          expect(logger.getLogs()).deep.equals([output[format]]);
+
+          expect((console as TestConsole)._stdout.write).toHaveBeenNthCalledWith(1, `${output[format]}\n`);
```

Comment:
The last `\n` added in expectation, which was earlier removed by `stubLoggerForProcessStd`.